examples/h2o/random_forest.ipynb

```diff
@@ -94,12 +94,13 @@
     "import mlflow\n",
```

Comment:
@ntakouris Do we need to make these formatting changes to the example notebooks? If not, can we revert them? Let me know if I'm missing something, but these don't appear to be related to the PR.

Reply:
My apologies, I thought I reverted that, but it certainly did not take effect. It's was a sideffect of using `black . `

Reply:
Got it! Can we revert these by checking out the notebooks from `master`?

Reply:
Yes, I have not yet pushed all the changes.